modules/microshift-nw-router-config-IPs.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-ip-addresses_{context}"]
+= Configuring router IP addresses
+
+You can restrict the network traffic to the router by configuring specific IP addresses. For example:
+* Use cases where the router is reachable only on internal networks, but not on northbound public networks.
+* Use cases where the router is reachable only by northbound public networks, but not on internal networks.
+* Use cases where the router is reachable by both internal networks and northbound public networks, but on separate IP addresses.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+//Q: Any other prereqs?
+
+.Procedure
+
+. Update the IP addresses in the `apiServer.advertiseAddress` field in the {microshift-short} `config.yaml` as shown in the following example:
```

Comment:
It is the `ingress.listenAddress` that needs update. I included some other parts of the config in the enhancement for context, but maybe we can drop them here? Include only the `ingress` section below?